sources/RuleTab.cpp

```diff
@@ -38,10 +41,10 @@ RuleTab::RuleTab()
 
 	if (fRuleList->CountItems() == 0) {
 		BAlert* alert = new BAlert("Filer",
-			"It appears that there aren't any rules for "
+			B_TRANSLATE("It appears that there aren't any rules for "
 			"organizing files. Would you like Filer to "
-			"add some basic ones for you?",
-			"No", "Yes");
+			"add some basic ones for you?"),
+			B_TRANSLATE("No"), B_TRANSLATE("Yes"));
 
```

Comment:
You have a point. How about: "Cancel" and "Add rules". Then it's obvious at a glance that "rules" are goint to be added.